From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Eelco Dolstra <eelco.dolstra@logicblox.com>
Date: Wed, 16 Apr 2014 10:59:28 +0200
Subject: [PATCH] Fix NixOS containers

In NixOS containers, the init script is bind-mounted into the
container, so checking early whether it exists will fail.
---
 src/nspawn/nspawn.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/nspawn/nspawn.c b/src/nspawn/nspawn.c
index e170958fc5..898a674631 100644
--- a/src/nspawn/nspawn.c
+++ b/src/nspawn/nspawn.c
@@ -5648,6 +5648,7 @@ static int run(int argc, char *argv[]) {
                                 goto finish;
                         }
                 } else {
+#if 0
                         _cleanup_free_ char *p = NULL;
 
                         if (arg_pivot_root_new)
@@ -5662,6 +5663,7 @@ static int run(int argc, char *argv[]) {
                                                     "Directory %s doesn't look like it has an OS tree (/usr/ directory is missing). Refusing.", arg_directory);
                                 goto finish;
                         }
+#endif
                 }
 
         } else {
